### PR TITLE
Add typings for moralis

### PR DESCRIPTION
**Pull request by rphansen91**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/moralis/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,1674 @@
...
+//                  Julien Quere <https://github.com/jlnquere>
+//                  Thibault MOCELLIN <https://github.com/tybi>
+//                  Raschid JF Rafaelly <https://github.com/RaschidJFR>
+//                  Jeff Gu Kang <https://github.com/jeffgukang>
+//                  Bui Tan Loc <https://github.com/buitanloc>
+//                  Linus Unnebäck <https://github.com/LinusU>
+//                  Jerome De Leon <https://github.com/JeromeDeLeon>
+//                  Kent Robin Haugen <https://github.com/kentrh>
+//                  Asen Lekov <https://github.com/L3K0V>
+//                  Switt Kongdachalert <https://github.com/swittk>
```
Comment (RaschidJFR) [inline types/moralis/index.d.ts:24]:
```suggestion
// Definitions by: 
```

**Review by RaschidJFR — CHANGES_REQUESTED** [review]
@rphansen91 It seems to me like you took this index from the [parse](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/parse) definition repo but you forgot to remove the reviewers, so we are getting linked to this conversation.

**Comment by rphansen91** [comment]
Thank you @RaschidJFR all Definitions by have been removed and replaced with myself

**Review by RaschidJFR — APPROVED** [review]

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Several questions/comments:

1. Can you explain why you need three different TS versions for this?
2. You have a `v1` directory but the version at the root says v0.0. The root version is supposed to be latest, while directories are supposed to contain older versions. But I don’t even see a v1 on npm?
3. > - [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.

    You checked this box but didn’t actually do what it said. Can you go back and make sure you’ve followed all the instructions and filled out the PR template accurately?

**Pull request closed without merging**